### PR TITLE
Export API types

### PR DESCRIPTION
**Pull request by tomas-zijdemans-vipps**

### CODE REVIEW

File: src/apis/types/recurring_types.ts [hunk 1]
```diff
@@ -349,7 +349,7 @@ export type ChargeEvent = {
 };
 
 /** Refund charge request */
-export type ModifyCharge = {
+export type ChargeModification = {
```
Comment (flinkgutt) [inline src/apis/types/recurring_types.ts:352]: Why not "ChargeBody"? Or is that leaking the abstraction too heavily in the name?

File: sample_code/redirectQR_sample.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import "https://deno.land/std@0.209.0/dotenv/load.ts";
+import { Client } from "../src/mod.ts";
```
Comment (flinkgutt) [inline sample_code/redirectQR_sample.ts:2]: Should we standardize on local version or do we need to first use the local, then update the dependency later to the deno.land url later?

File: src/apis/types/recurring_types.ts (hunk 1 above)
Comment (tomas-zijdemans-vipps) [reply src/apis/types/recurring_types.ts:352]: Hmm. Looking at EPayment, the equivalent is "EPaymentModificationRequest". I'll update the types here to be consistent with EPayment

File: sample_code/redirectQR_sample.ts (hunk 2 above)
Comment (tomas-zijdemans-vipps) [reply sample_code/redirectQR_sample.ts:2]: Oooooops!!! That should be deno.land. Will fix!

**Pull request merged**